development/sentry-publish.js

```diff
@@ -109,17 +108,6 @@ async function checkIfVersionExists(version) {
   );
 }
 
-async function checkIfVersionHasArtifacts(version) {
```

Comment:
From local testing, it seems that this was not working. It was returning empty even for releases with artifacts. I think this is a legacy command that no longer works due to a change in how Sentry tracks artifacts that was made >1 year ago (they're no longer considered as "release bundles", they're "artifact bundles" now, which aren't returned here.)

Since it wasn't working anyway, I removed it rather than trying to update it to use `--dist` correctly (it didn't seem to accept a `--dist` flag, as far as I could tell, so there was no easy way to get this to work).